.github/workflows/build-docker.yml

```diff
@@ -11,9 +11,15 @@ jobs:
       GROCY_IMAGE_TAG: ${{ github.event.release.tag_name }}
     steps:
     - uses: actions/checkout@v2
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
     - uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
-    - run: docker-compose build
-    - run: docker-compose push
+    - name: Build and push
+      uses: docker/bake-action@master
+      with:
+        set: "*.platform=linux/amd64,linux/arm64,linux/arm/v7,linux/arm/v6"
```

Comment:
How did you choose the list of platforms bake, out of interest?

Reply:
I did a quick test on a raspberry pi I have. Getting `exec format error`. Gotta make sure I'm using the correct arch image. Something in the build might also be arch specific. I'll have to investigate. However, I can confirm that this will create multi-arch images. Check out [this](https://hub.docker.com/r/rickcoxdev/grocy/tags?page=1&ordering=last_updated).

Reply:
Thanks!